apps/demos/Demos/Stepper/Overview/description.md

```diff
@@ -1,5 +1,40 @@
-Our upcoming major release (v25.1) will introduce a Stepper component designed to describe/detail progress as users navigate through a defined sequence. 
+The DevExtreme Stepper is a UI component that enables a user to navigate multi-step processes like purchases and forms. Stepper supports multiple customization options.
+
+To get started with DevExtreme Stepper, refer to the following tutorial for step-by-step instructions: [Getting Started with Stepper](/Documentation/Guide/UI_Components/Stepper/Getting_Started_with_Stepper/).
 <!--split-->
-Stepper will work especially well for multi-step processes like filling forms, onboarding flows, or wizards.   
 
-You will be able to customize the appearance of individual steps and modify component settings to address specific usage scenarios. 
\ No newline at end of file
+## Base Configuration
+
+The Stepper includes the following base properties:
+
+- [orientation](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/#orientation) (default: *'horizontal'*)
+The Stepper's orientation.
+- [linear](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/#linear) (default: `true`)
+Specifies whether users navigate the Stepper sequentally or skip steps.
+- [selectOnFocus](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/#selectOnFocus) (default: `true`)
+Specifies whether steps focused with keyboard arrows are automatically selected or not.
+- [rtlEnabled](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/#rtlEnabled) (default: `false`)
+Switches the Stepper to a right-to-left mode.
+
+## Steps
+
+To specify Stepper steps, define [items](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/) or [dataSource](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/#dataSource). Define **dataSource** if you need to utilize remote or processed data. 
+
+You can assign the following properties to each step:
+
+- [text](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#text)
+Indicator text. If you do not define this option, Stepper will number steps sequentially.
+- [icon](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#icon)
+Indicator icon. Stepper prioritizes icons over the **text** property.
+- [label](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#label)
+Step caption.
+- [optional](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#optional)
+Adds an *(Optional)* sub-caption.
+- [isValid](/Documentation/ApiReference/UI_Components/dxStepper/Configuration/items/#isValid)
+A visual validation indicator.
```

Comment:
Sometimes you use a noun to describe the property: `caption` - "Step caption displayed ...". This makes sense, because the property value actually holds that caption  string. 

Sometimes you use a verb: `optional` - "Adds a label ... ". Also makes sense, because the property holds `true` or `false`, but the step does something based on this value.  

But in this one, the property still holds `true` or `false`, but you describe it as "A visual indicator". The property doesn't hold an indicator. It holds a value that specifies whether an indicator will appear. 

```suggestion
Allows you to indicate that user input on this step results in validation errors.
```